### PR TITLE
[DELTA-35] Add BUSD for polygon and bnb bridge

### PR DESCRIPTION
**Pull request by deltafilzr**
added
1. logos
2. configuration file for Polygon and BNB

### CODE REVIEW

File: src/config/bridge/fullDeployConfig.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "polygon": {
```
Comment (pjxthy) [inline src/config/bridge/fullDeployConfig.json:2]: remove all token logos with wh eg public/cryptologos/BNB_wh.png. we do not usually make the difference in logo for wrapped tokens.

also cryptologos -> logos.

File: src/config/bridge/fullDeployConfig.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    "chain": {
+      "wormholeChainId": "CHAIN_ID_POLYGON",
+      "name": "Polygon",
+      "logoURI": "https://github.com/deltafi-trade/deltafi-app-v2/blob/main/public/cryptologos/POLYGON.svg"
+    },
+    "mainnet-prod": {
+      "tokenInfoList": {
+        "source": [
+          {
+            "name": "Wrapped Matic",
```
Comment (pjxthy) [inline src/config/bridge/fullDeployConfig.json:12]: please be specific on what wrapped tokens. there are a lot of wrapped tokens like wormhole wrapped etc.

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: src/config/bridge/fullDeployConfig.json (hunk 1 above)
Comment (deltafilzr) [reply src/config/bridge/fullDeployConfig.json:2]: done!

File: .prettierignore [hunk 3]
```diff
@@ -1,4 +1,3 @@
-*.json
```
Comment (pjxthy) [inline .prettierignore:1]: why remove json here?

File: src/config/bridge/fullDeployConfig.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
+{
+  "polygon": {
+    "chain": {
+      "wormholeChainId": "CHAIN_ID_POLYGON",
```
Comment (pjxthy) [inline src/config/bridge/fullDeployConfig.json:4]: i do not think this config is correct.

1. each chain's config should be separate
2. matic token is redundant
3. please remove ploygon and bsc in the toke names. chain param should be a separate one.

**Review by pjxthy — CHANGES_REQUESTED** [review]

**Review by pjxthy — APPROVED** [review]

**Pull request merged**